<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SumPrice</class>
 <widget class="QWidget" name="SumPrice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SumPrice</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>421</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>QLineEdit输入和显示数值</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>421</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="numlabel">
            <property name="text">
             <string>数量</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="num"/>
          </item>
          <item>
           <widget class="QLabel" name="pricelabel">
            <property name="text">
             <string>单价</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="price"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="CalculateButton">
            <property name="text">
             <string>点击计算</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="sumpricelabel">
              <property name="text">
               <string>总价</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="sumprice"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>SpinBox输入和显示数值</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>421</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="numSpinBox">
            <property name="text">
             <string>数量</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="numspinBox">
            <property name="suffix">
             <string> kg </string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="singleStep">
             <number>0</number>
            </property>
            <property name="stepType">
             <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="priceSpinBox">
            <property name="text">
             <string>单价</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="pricedoubleSpinBox">
            <property name="prefix">
             <string>$ </string>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="autocalculateSpinBox">
            <property name="text">
             <string>自动计算总价</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="autocalpricedoubleSpinBox">
            <property name="prefix">
             <string>$ </string>
            </property>
            <property name="minimum">
             <double>50.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
